brave-test/src/main/java/brave/extension/util/KafkaUtil.java

```diff
@@ -169,4 +159,28 @@ public static List<Entry<String, String>> loadEvents(String resourceFile) throws
 
         return kafkaEvents;
     }
+
+    /*
+    * Consume a number of events from topic
+    * */
+    public static List<ConsumerRecord<String, String>> consumeEvents(Consumer<String, String> consumer, int eventCount) throws InterruptedException {
```

Comment:
Nếu là "Consume a number of events from topic" thì interface phải là 
```
consumeEvents(String topicName, int eventCount, Duration timeout) {{
...
}
```
mới phải. Em có thể chủ động khởi tạo consumer trong này. 

Tạm thời em giữ hàm này. Nhưng đặt trong KafkaStub nhé, là 1 phần của interface cung cấp ra cho việc viết tests.